### ISSUE TITLE
Code does not properly support vs script pathing.

### ISSUE THREAD

**Issue by smaudet** [pavelliavonau/cmakeconverter#118]
**Describe the bug**
e.g. copy $(SolutionDir)..\..\file.txt does not properly get parsed, leading to corrupted CMakeLists.txt

**To Reproduce**
Use windows pathing in a vcxproj

**Expected behavior**
This should work.

**Additional context**
Additionally this seems to cause a cascade of other issues, such as VS variables not being replaced.

**Comment by pavelliavonau** [comment]
Pathing like this works. But there might be exceptions of this rule. With my cases it looks fine.
What case exactly do you have?

**Comment by pavelliavonau** [comment]
@smaudet , is it from build step/event?

**Comment by smaudet** [comment]
@pavelliavonau yes, it is - I'll try to post an example later but my use case has something like so:

build/nested/dir
actual/src/dir

Then many, many projects in the nested project dir which have pre/post build events and reference stuff in actual/src/dir...

...it took me two days to manually reparse everything that was wrong (vs solution dirs, scripts with different functionality versus what you would expect, wrong directories - on top of moving CMakeLists to the source directory because some ides expect things to work that way...

I got some patches half working but I don't have a ton of time, I got it half all fixed, but figured I should document my woes.
